kf/bpf.go

```diff
@@ -975,9 +985,13 @@ func (b *BPF) VerifyPinnedMapVanish(chain bool) error {
 
 	var err error
 	log.Debug().Msgf("VerifyPinnedMapVanish : Program %s MapName %s", b.Program.Name, b.Program.MapName)
+	path := b.MapFullPath()
+	if strings.Contains(path, "..") {
```

Comment:
This doesn't seem like the right place to validate config values for b.hostConfig.BpfMapDefaultPath and b.Program.MapName.  What about doing validation checks in kf/nfconfig.go and removing the ones here?

Reply:
if we remove this check github check is showing security issue